app/controllers/scientist_experiments_controller.rb

```diff
@@ -0,0 +1,7 @@
+class ScientistExperimentsController < ApplicationController
+  def destroy
+    @scientist_experiment = ScientistExperiment.find(params[:id])
+    @scientist_experiment.destroy
+    redirect_to(scientist_path(@scientist_experiment.scientist_id))
+  end
```

Comment:
US2 could have been done much more easily if you had been using nested routing! Because you weren't, you had to jump through to many hoops to get to what you wanted.